### DISCUSSION TITLE
My Feature Suggestions

### DISCUSSION THREAD

**Discussion by vistabunny** [stoatchat/discussions#752] [Feature Suggestions]
here's my feature suggestions list I want for this new Revolt Client:

- Gradient Roles (Like Guilded)
- Server Groups (like Guilded)
- Transfer Server Ownership (You could transfer ownership to another user to handle the server)
- Self-Assign Roles
- Role Mentions (like what Discord and Guilded has)

**Comment by bitrain574** [comment]
That’ll be great.

**Comment by Rexogamer** [comment]
just a few notes:
- gradient roles already exist, though currently you need to use the API to set them
- in the meantime, ownership transfers are doable by support
- also in the meantime, self assigns are currently doable via the Roles bot on Discover
- role mentions are also planned, there's an issue somewhere for this

**Reply by vistabunny** [reply]
but however... API needs to be required.. but I wish Revolt would make an update to where Gradient Roles would be available to the public, and without waiting for support to wait for an ownership transfer, I hope there would be a button next to "Save" that says "Transfer Ownership"
